orchestra/utils/view_helpers.py

```diff
@@ -12,3 +14,82 @@ class IsAssociatedWorker(permissions.BasePermission):
     def has_object_permission(self, request, view, obj):
         worker = Worker.objects.get(user=request.user)
         return obj.worker == worker
+
+
+def get_changed_fields(old_todo, new_todo):
+    # To avoid Slack noise, only care about title and details
+    changed_fields = []
+    if old_todo.title != new_todo.title:
+        changed_fields.append('title')
+    if old_todo.details != new_todo.details:
+        changed_fields.append('details')
+    if len(changed_fields) > 1:
+        changed_fields[-1] = 'and {}'.format(changed_fields[-1])
+    return ', '.join(changed_fields)
```

Comment:
The serial comma gets added for the third item. Sorry about having to fix tests! :/
```suggestion
    return ' '.join(changed_fields)
```